### PR TITLE
Make awesome books using Es6

### PR DESCRIPTION
**Pull request by zunairkhan811**
The use of the following things has been added to the project.
- luxon library has been added and Datetime is updated accordingly.
- Modules directory is made and the module file is made.
- No linter errors

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-alert */
+/* eslint-disable comma-dangle */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```
Comment (wayungi) [inline main.js]: - [x] Please do not disable linters. Linters help programmers to write code of the same quality and standard. They promote good coding practice 🔖

File: main.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  static removebook(title) {
+    const books = store.getbooks();
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```
Comment (wayungi) [inline main.js]: - [x] Please use modules as instructed in the requirements.  Split the main.js into the following modules 🥲 
   - book.js ✔️ 
   - ui.js ✔️ 
   - store.js ✔️ 
   
   The instructions are laid dow below 👇🏾 
   
mod

File: main.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+  static removebook(title) {
+    const books = store.getbooks();
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```
Comment (wayungi) [inline main.js]: - [x] Please use arrow functions for all your functions. This is quite easy to implement. The syntax as is follows `const removebook = (title) => {//write function body here}` 😀

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @zunairkhan811 🤝 , 
image

Good job so far ✔️!
There are some issues that you still need to work on to go to the next project but you are almost there! 👍🏾 

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body 

## Required Changes ♻️
_Check the comments under the review._
- [x] Please follow the instructions given in the assignment. You need to create an index.js and index.css as requested in the exercise.

ex


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @zunairkhan811,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**